packages/graphql/src/types/neo4j-graphql-subscriptions-context.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Neo4jGraphQLCommonContext } from "./neo4j-graphql-common-context";
+
+export interface Neo4jGraphQLSubscriptionsConnectionParams extends Neo4jGraphQLCommonContext {}
```

Comment:
Isn't it a bit redundant to call the superclass Neo4jGraphQL**Common**Context?

Reply:
Well `Neo4jGraphQLContext` is the class name we expose to users, so we can't use that, do you want me to call it `INeo4jGraphQLContext`? 😜

Reply:
Good point. What about `Neo4jGraphQLContextInterface`?

I know it is not the bestest name ever, but I think still beats slightly `Common`